interface/patient_file/summary/demographics.php

```diff
@@ -169,7 +169,10 @@ function image_widget($doc_id,$doc_catg)
 
 <head>
 <?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
Should also stick the following here:
``````
<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
<?php } ?>
``````